src/CameraControls.ts

```diff
@@ -2946,64 +2946,64 @@ export class CameraControls extends EventDispatcher {
 
 	}
 
-}
+	static createBoundingSphere( object3d: _THREE.Object3D, out: _THREE.Sphere = new THREE.Sphere() ): _THREE.Sphere {
```

Comment:
Here, I've set up a default for `out`: `= new THREE.Sphere()`

Reply:
okay👍